File: cell.cpp
#include "cell.h"

Cell::Cell(QChar let, QObject *parent) :
    QObject(parent)
{
    letter_ = let;
    isMarked_ = false;
}

QChar Cell::getLetter() const {
    return letter_;
}
void Cell::setLetter(QChar c){
    letter_ = c;
}

bool Cell::isMarked() const {
    return isMarked_;
}

void Cell::setMarked(bool value) {
    isMarked_ = value;
}

File: src/Cell.py
__author__ = 'user1'
from PySide import QtCore


class Cell(QtCore.QObject):
    def __init__(self, letter):
        super(Cell, self).__init__()
        self.__letter__ = letter
        self.__is_marked__ = False

    def get_letter(self):
        return self.__letter__

    def set_letter(self, letter):
        self.__letter__ = letter

    def is_marked(self):
        return self.__is_marked__

    def set_marked(self, value):
        self.__is_marked__ = value